app/_src/gateway/production/networking/troubleshoot-tls.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Troubleshoot TLS
+content_type: reference
+---
+
+## PostgreSQL
+
+### Could not accept SSL connection:
+
+If you are receiving either of the following error messages, you are experiencing an issue with mismatched SSL versions.
+
+```sh
+LOG:  could not accept SSL connection: wrong version number
+HINT:  This may indicate that the client does not support any SSL protocol version between TLSv1.1 and TLSv1.1.
+```
+
+```sh
+LOG:  could not accept SSL connection: unsupported protocol
+HINT:  This may indicate that the client does not support any SSL protocol version between TLSv1.2 and TLSv1.3.
+```
+
+In this case the client is trying to connect to the server using a version of the TLS protocol that is not supported by the server. Check what version PostgreSQL supports and adjust the client side TLS version. 
+
+You can check the TLS version by running the `pg_config --configure` command. 
+
+Kong recommends TLSv1.2 or higher because lower versions are deprecated and considered unsecure.
+
+### Could not load server certificate file
+
+If you are receiving this error message: 
+
+```sh
+FATAL:  could not load server certificate file "server.crt": No such file or directory
+```
+The server is unable to find the certificate file. By default, the certificate file should be named `server.crt`, and it should be placed in the server's data directory. Other names are allowed, but they should be explicitly specified in the `ssl_cert_file`. 
+The combined method combines any authentication method in the `hostssl` entry with the certification of a client certificate using the `clientcert` authentication option. 
```

Comment:
What's the combined method and why is it relevant here?

Reply:
I think that's got copied in from when I split the file.